docs/Explore Algorithms/AI Services/Quickstart - Creare a Visual Search Engine.ipynb

```diff
@@ -46,31 +50,31 @@
     "    .withColumn(\"Tags\", split(col(\"Tags\"), \",\").cast(\"array<string>\"))\n",
     "    .limit(25)\n",
     ")"
-   ],
-   "outputs": [],
-   "metadata": {
-    "collapsed": true
-   }
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
    "source": [
     "<img src=\"https://mmlspark.blob.core.windows.net/graphics/CognitiveSearchHyperscale/MetArtworkSamples.png\" width=\"800\" style=\"float: center;\"/>"
-   ],
-   "metadata": {}
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": 7,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
    "source": [
     "from synapse.ml.cognitive import AnalyzeImage\n",
     "from synapse.ml.stages import SelectColumns\n",
     "\n",
     "# define pipeline\n",
     "describeImage = (\n",
     "    AnalyzeImage()\n",
-    "    .setSubscriptionKey(cognitive_key)\n",
-    "    .setLocation(cognitive_loc)\n",
+    "    .setSubscriptionKey(ai_service_key)\n",
+    "    .setLocation(ai_service_loc)\n",
```

Comment:
We should make sure these demos work with a brand new ai service key as urls and whatnot in these services might need to be changed